lib/dradis/plugins/templates.rb

```diff
@@ -28,7 +46,32 @@ def copy_templates(args={})
           end
         end
 
-        def plugin_templates(args={})
+        def migrate_templates_to_mappings(args = {})
+          # return if the integration doesn't provide any templates ex. projects, cve
+          return unless paths['dradis/templates'].existent.any?
```

Comment:
Since there's only one type of file we care about (.template), and you rename them (to .legacy), there're going to be files even when we've processed this once already. Perhaps adjusting the search for ".template" files, or return if none?

Reply:
This part will ensure we skip integrations without a `templates/` dir at all ex. projects, cve
I have a check a little further down that returns if there are no .template files in the dir